lib/src/modules/protos/macho.proto

```diff
@@ -10,13 +10,63 @@ option (yara.module_options) = {
   rust_module: "macho"
 };
 
+message MinVersion {
+  optional uint32 device = 1;
```

Comment:
If values in this field are one of the values in the `DEVICE_TYPE` enum, make this field of type `DEVICE_TYPE`. This way it will be shown in the YAML output as `device: MACOSX` instead of `device: 36`.

Reply:
Fixed 😄